### PR TITLE
tg-can-1225

### PR DESCRIPTION
**Pull request by nalwa**
this pR has fixes fro can-1225 and can-1237

@akuks  please review and merge.

### CODE REVIEW

File: app/Helpers/TopicSupport.php [hunk 1]
```diff
@@ -16,7 +16,11 @@
 use App\Events\PromotedDelegatesMailEvent;
 use App\Facades\GetPushNotificationToSupporter;
 use App\Events\NotifyDelegatedAndDelegatorMailEvent;
+<<<<<<< HEAD
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Remove these lines. It will cause the application to fail

File: app/Helpers/TopicSupport.php [hunk 2]
```diff
@@ -16,7 +16,11 @@
...
+use App\Facades\util;
+=======
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Remove this lines. It will cause the application to fail

File: app/Helpers/TopicSupport.php [hunk 3]
```diff
@@ -16,7 +16,11 @@
...
 use App\Facades\Util;
+>>>>>>> development
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Remove this lines. It will cause the application to fail

**Review by akuks — CHANGES_REQUESTED** [review]
Please do the requested changes

File: app/Helpers/TopicSupport.php (hunk 3 above)
Comment (nalwa) [reply app/Helpers/TopicSupport.php]: @akuks  for this PR i am not getting any conflict on my local on this branch even after merging development to it.

**Review by akuks — APPROVED** [review]

**Pull request merged**